tests/tensors/test_ckks_vector.py

```diff
@@ -1,5 +1,6 @@
 import tenseal as ts
 import pytest
+import numpy as np
```

Comment:
Since you moved `numpy` import here, I thing we should remove the import from other test functions, `test_vec_plain_matrix_mul` function still have import `numpy` there (line 375).